jetty-websocket/websocket-javax-client/src/main/java/org/eclipse/jetty/websocket/javax/client/JavaxWebSocketClientContainerProvider.java

```diff
@@ -50,31 +51,43 @@ public static void stop(WebSocketContainer container) throws Exception
      * </p>
      */
     @Override
-    protected WebSocketContainer getContainer()
+    public WebSocketContainer getContainer()
```

Comment:
Why did you open it from `protected` to `public`?

Reply:
Because with the new method we are setting the precedent that you can call these methods directly on the `JavaxWebSocketClientContainer` and not always go though the ServiceLoader using `ContainerProvider.getWebSocketContainer()`.